src/loading/loading.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input } from "@angular/core";
+
+@Component({
+	selector: "ibm-loading",
+	template: `
+		<div class="bx--loading-overlay">
```

Comment:
My apologies - My "overlay" comment seems to have introduced a confusion. Overlay is an _optional_ element, that comes only with non-small spinner.